src/Twitter.php

```diff
@@ -1371,13 +1385,17 @@ public function init(string $path, Http\Client $client) : void
      * returned verbatim. Otherwise, a zero is returned.
      *
      * @param string|int $int
-     * @return string|int
      */
-    protected function validInteger($int)
+    protected function validInteger($int) : int
     {
+        if (! is_string($int) && ! is_int($int)) {
```

Comment:
Some Twitter API identifiers exceed PHP_INT_MAX, which is why a regex is performed here; the value may very well be a string.